source/AI.cpp

```diff
@@ -549,19 +549,17 @@ void AI::Step(Command &activeCommands)
 	step = (step + 1) & 31;
 	int targetTurn = 0;
 	int minerCount = 0;
-	const int maxMinerCount = minables.empty() ? 0 : 9;
-	bool opportunisticEscorts = !Preferences::Has("Turrets focus fire");
-	bool fightersRetreat = Preferences::Has("Damaged fighters retreat");
-	const int npcMaxMiningTime = GameData::GetGamerules().NPCMaxMiningTime();
+	const bool playerOpportunisticEscorts = !Preferences::Has("Turrets focus fire");
+	const bool playerFightersRetreat = Preferences::Has("Damaged fighters retreat");
```

Comment:
I  think all these variable renames are useless. The names aren't bad enough to justify it imo.